server/sequelize/config/config.js

```diff
@@ -0,0 +1,47 @@
+const dotenv = require("dotenv");
+dotenv.config();
+
+module.exports = {
+  development: {
+    username: process.env.DB_USERNAME,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_DATABASE,
+    host: process.env.DB_HOST,
+    dialect: process.env.DB_DIALECT,
+    port: process.env.DB_PORT,
+    dialectOptions: {
+      ssl: {
+        require: true,
+        rejectUnauthorized: false,
+      },
+    },
```

Comment:
Does everuone on the team have a database running locally that needs SSL? Can you check with the team